example/src/test/resources/webcontrol/pipes/ShowConfigurationStatus_all.xml

```diff
@@ -8,7 +8,7 @@
   </configurations>
   <registeredAdapters all="true" log="false">
       <configurationMessages>
-         <configurationMessage date="0000-00-00 00:00:00.000" level="INFO">Configuration [IAF_WebControl] LiquiBase applied [6] change(s) and added tag [IAF_Util:6:Niels Meijer]</configurationMessage>
+         <configurationMessage date="0000-00-00 00:00:00.000" level="INFO">Configuration [IAF_WebControl] LiquiBase applied [7] change(s) and added tag [IAF_Util:7:Ali Sihab Akcan]</configurationMessage>
```

Comment:
I'm assuming this is the only change in this (massive) file. If so, could you revert the other changes? The test performs an XML diff so it doesn't look at the order of the elements.